.circleci/config.yml

```diff
@@ -143,7 +143,8 @@ jobs:
     steps:
       - attach_workspace:
           at: ~/
-
+      - setup_remote_docker
+      - run: ../../_scripts/install_docker.sh
```

Comment:
or just `run: docker network create fxa-net || true` so we don't pull a bunch of images we don't need?

Reply:
Don't we end up needing those images anyway? pm2 runs redis, memcache, etc